app/javascript/components/rooms/RoomsList.jsx

```diff
@@ -1,29 +1,33 @@
-import React from 'react';
+import React, { useState } from 'react';
 import {
-  Row, Col, Button, Spinner, Form, FormControl,
+  Row, Col, Form, FormControl,
 } from 'react-bootstrap';
+import Spinner from '../shared/stylings/Spinner';
 import RoomCard from './RoomCard';
 import useRooms from '../../hooks/queries/rooms/useRooms';
 import ButtonLink from '../shared/stylings/buttons/ButtonLink';
 
 export default function RoomsList() {
   const { isLoading, data: rooms } = useRooms();
+  const [search, setSearch] = useState('');
   if (isLoading) return <Spinner />;
 
   return (
     <>
       <Row className="mt-4 mb-4">
         <Col>
-          {/* TODO: May need to change search form when implementing search functionality */}
-          <Form className="d-flex">
+          <Form>
             <FormControl
```

Comment:
I think it's better to use a fully [uncontrolled ](https://reactjs.org/docs/uncontrolled-components.html) form rather then a fully [controlled](https://reactjs.org/docs/forms.html#controlled-components) one.
Also, we're using the [react-hook-form](https://react-hook-form.com/) so it's advised to use it even for such simple form for consistency, maintainability and performance.

Reply:
NOTE: We discussed this and decided to leave it as is